### DISCUSSION TITLE
React JS Community

### DISCUSSION THREAD

**Discussion by prasadbobby** [prasadbobby/portfolio#5] [Q&A] [answered]
Any best resources to learn React JS??

**Comment by mahendrasaikumargandham** [comment, marked as answer]
Clever Programmer is a Youtube Channel where you can find best Resource. @prasadbobby 
Here is the Link:
https://www.youtube.com/c/CleverProgrammer

**Comment by IgorSwatowski** [comment]
I suggest @prasadbobby this website.

https://frontendmasters.com/
